analysis_runner/analysisrunner.py

```diff
@@ -52,6 +60,7 @@ def run_analysis_runner(
     commit=None,
     repository=None,
     cwd=None,
+    environment_variables=None,
```

Comment:
This would then become:

```suggestion
    environment_variables: List[str]=None,
```